src/sidebar/components/thread.js

```diff
@@ -40,7 +40,7 @@ function Thread({ showDocumentInfo = false, thread, threadsService }) {
     child => countVisible(child) > 0
   );
 
-  const onToggleReplies = () => setCollapsed(thread.id, !thread.collapsed);
+  const onToggleReplies = () => setExpanded(thread.id, !!thread.collapsed);
```

Comment:
```suggestion
  const onToggleReplies = () => setExpanded(thread.id, !!thread.collapsed);
```

Is the `!!` needed here? If we can be sure that `thread.collapsed` is actually a boolean then it shouldn't be.